optionals/bnae_rk95_compat_ace/CfgWeapons.hpp

```diff
@@ -0,0 +1,99 @@
+class Mode_SemiAuto;
+class Mode_FullAuto;
+class asdg_OpticRail;
+class asdg_OpticRail1913: asdg_OpticRail {};
+
+class CfgWeapons {
+    class Rifle;
+    class Rifle_Base_F: Rifle {
+        class WeaponSlotsInfo;
+    };
+
+    // http://guns.connect.fi/gow/M95.html
+    class bnae_rk95_base: Rifle_Base_F {
+        ACE_barrelTwist = 240.0;
+        ACE_barrelLength = 420.0;
+        class Single: Mode_SemiAuto {
+            dispersion = 0.00044; // 1.5 MOA; default: 0.00058 = 2 MOA
+        };
+        class FullAuto: Mode_FullAuto {
+            dispersion = 0.00044; // 1.5 MOA; default: 0.00058 = 2 MOA
+        };
+    };
+
+    class bnae_rk95_virtual: bnae_rk95_base {};
```

Comment:
```suggestion
    class bnae_rk95_virtual;
```
Just this should work, less chance to break inheritance.